lib/suse/connect/api.rb

```diff
@@ -60,36 +60,17 @@ def list_subscriptions
         make_paginated_request(:get, "#{connect_api}/organizations/subscriptions")
       end
 
-      def forward_system_activations(system)
-        params = {
-          login: system.login,
-          password: system.password,
-          hostname: system.hostname,
-          regcodes: [],
-          products: generate_product_listing_for(system),
-          hwinfo: generate_hwinfo_for(system),
-          last_seen_at: system.last_seen_at
-        }
-
-        make_single_request(
-          :post,
-          "#{connect_api}/organizations/systems",
-          { body: params.to_json }
-        )
-      end
-
       def send_bulk_system_update(systems, system_limit = nil)
         system_limit ||= BULK_SYSTEM_REQUEST_LIMIT
-        last_response = nil
         systems.each_slice(system_limit) do |batched_systems|
           params = prepare_payload_for_bulk_update(batched_systems)
-          last_response = make_single_request(
+          response = make_single_request(
             :put,
             "#{connect_api}/organizations/systems",
             { body: params.to_json }
           )
+          yield response
```

Comment:
Does this work to `yield` in a loop so that the receiving block receives all yielded results concatenated? Wow! :-)

Reply:
Yes it does! Didn't know about it either!

Reply:
:+1: